shared/actions/fs.js

```diff
@@ -267,11 +267,7 @@ function* pollSyncStatusUntilDone(): Saga.SagaGenerator<any, any> {
 }
 
 function _setupFSHandlers() {
-  return Saga.put((dispatch: Dispatch) => {
-    engine().setIncomingHandler('keybase.1.NotifyFS.FSSyncActivity', ({status}) => {
-      dispatch(FsGen.createFsActivity())
-    })
-  })
+  engine().setIncomingActionCreators('keybase.1.NotifyFS.FSSyncActivity', () => [FsGen.createFsActivity()])
```

Comment:
to test this you can just do something in kbfs like copy a file into a folder that should trigger it